plugins/outputs/wavefront/README.md

```diff
@@ -5,8 +5,7 @@ This plugin writes to a [Wavefront](https://www.wavefront.com) proxy, in Wavefro
 ## Configuration
 
 ```toml
-  ## Url for Wavefront Direct Ingestion or using HTTP with Wavefront Proxy
-  ## If using Wavefront Proxy, also specify port. example: http://proxyserver:2878
+  ## Url for Wavefront Direct Ingestion.
```

Comment:
Is a port not required for a proxy?

Reply:
Port is required for a proxy, see below, but the port is not required for direct ingestion.

Reply:
ok, but why are you removing this from the docs?

Reply:
@powersj we had a customer run into an issues using Telegraf with Wavefront Proxy – when we asked them to switch to the Proxy-specific config in Telegraf (`host` and `port`), it resolved their issue. We want to direct future users to that config option.

@keep94 I think we might update the comment in the commit to something like:

`## Url for Wavefront Direct Ingestion. For Wavefront Proxy Ingestion, see the 'host' and 'port' options below.`

Reply:
@oppegard thank you for the background, that helps greatly. I agree with your suggestion and would like to see that change made.

Reply:
oppegard's suggestion applied to this PR.